GVFS/GVFS.Virtualization/Projection/GitIndexProjection.LazyUTF8String.cs

```diff
@@ -183,11 +188,26 @@ public unsafe int CaseInsensitiveCompare(LazyUTF8String other)
                 return this.length - other.length;
             }
 
+            public unsafe int CaseInsensitiveCompare(LazyUTF8String other)
+            {
+                return this.Compare(other, false);
+            }
+
+            public unsafe int CaseSensitiveCompare(LazyUTF8String other)
+            {
+                return this.Compare(other, true);

```

Comment:
Style nit:  Here and above can you use a named parameter for the bools?

*Example*

`return this.Compare(other, caseSensitive: true);`

Reply:
Definitely -- will do in next set of updates!